File: tools/debug_search_v2.py
import sys

file_path = 'c:/Users/DELL/Downloads/portel-master (2)/portel-master/client/index.html'

def search_strings(targets):
    try:
        with open(file_path, 'r', encoding='utf-8') as f:
            for i, line in enumerate(f, 1):
                for target in targets:
                    if target in line:
                        print(f"FOUND '{target}' at line {i}: {line.strip()[:100]}")
    except Exception as e:
        print(f"Error: {e}")

if __name__ == "__main__":
    search_strings(['<script', 'StudentGradingView', 'const showCamera = captureOnly;'])

File: debug_search_v2.js
const fs = require('fs');
const path = 'c:/Users/DELL/Downloads/portel-master (2)/portel-master/client/index.html';

const targets = ['<script', 'StudentGradingView', 'const showCamera = captureOnly;'];

try {
    const content = fs.readFileSync(path, 'utf8');
    const lines = content.split('\n');
    lines.forEach((line, i) => {
        const lineNum = i + 1;
        targets.forEach(target => {
            if (line.includes(target)) {
                console.log(`FOUND '${target}' at line ${lineNum}: ${line.trim().substring(0, 100)}`);
            }
        });
    });
} catch (err) {
    console.error(err);
}
